package.json

```diff
@@ -44,10 +44,13 @@
     "cheerio": "0.22.0",
     "colors": "1.1.2",
     "commander": "2.9.0",
+    "emoji-name-map": "^1.2.2",
     "event-stream": "3.3.4",
     "express": "4.14.1",
     "fs-extra": "2.0.0",
     "irc-framework": "2.5.5",
+    "jquery-textcomplete": "^1.8.0",
+    "json-loader": "^0.5.4",
```

Comment:
Yep.

I also don't think `json-loader` is even needed here as `emoji-name-map` is used.